src/profile-logic/tracks.js

```diff
@@ -624,29 +631,36 @@ function _isThreadIdle(profile: Profile, thread: Thread): boolean {
 }
 
 function _isContentThreadWithNoPaint(thread: Thread): boolean {
-  // Hide content threads with no RefreshDriverTick. This indicates they were
-  // not painted to, and most likely idle. This is just a heuristic to help users.
   if (thread.name === 'GeckoMain' && thread.processType === 'tab') {
-    let isPaintMarkerFound = false;
-    if (thread.stringTable.hasString('RefreshDriverTick')) {
-      const paintStringIndex = thread.stringTable.indexForString(
-        'RefreshDriverTick'
-      );
+    return _isThreadWithNoPaint(thread);
```

Comment:
Diff looks complicated but I actually only removed the content of `_isContentThreadWithNoPaint` and moved it to `_isThreadWithNoPaint` so we can use it in other places.